doc/api/modules.md

```diff
@@ -175,15 +175,15 @@ LOAD_AS_DIRECTORY(X)
 2. LOAD_INDEX(X)
 
 LOAD_NODE_MODULES(X, START)
-1. let DIRS=NODE_MODULES_PATHS(START)
+1. let DIRS = NODE_MODULES_PATHS(START)
 2. for each DIR in DIRS:
    a. LOAD_AS_FILE(DIR/X)
    b. LOAD_AS_DIRECTORY(DIR/X)
 
 NODE_MODULES_PATHS(START)
 1. let PARTS = path split(START)
 2. let I = count of PARTS - 1
-3. let DIRS = []
+3. let DIRS = [ GLOBAL_FOLDERS ]
```

Comment:
The term “GLOBAL_FOLDERS” doesn’t appear to be defined anywhere; if you mean the list in line 424, could you alter that list as well to use this identical term?

Ideally, this term would also link down to that section.

Reply:
@ljharb What would be the best way to capture this?  The main heading is regarding "loading from the global folders."  Would it make sense to say `Additionally, Node.js will search in the global folders`?

Reply:
Even just a sentence in the below section like “This list of GLOBAL_FOLDERS:” or something

Reply:
It’d be cool if the mention in the algorithm itself could also be linked.